common/src/main/scala/com/gu/zuora/ZuoraService.scala

```diff
@@ -26,12 +28,26 @@ class ZuoraService(config: ZuoraConfig, client: FutureHttpClient, baseUrl: Optio
   def getAccount(accountNumber: String): Future[GetAccountResponse] =
     get[GetAccountResponse](s"accounts/$accountNumber")
 
+  def getAccountIds(identityId: String): Future[List[String]] = {
+    val queryData = QueryData(s"select AccountNumber from account where IdentityId__c = '$identityId'")
+    post[QueryResponse](s"action/query", queryData.asJson).map(_.records.map(_.AccountNumber))
+  }
+
+  def getSubscriptions(accountId: String): Future[List[Subscription]] =
+    get[SubscriptionsResponse](s"subscriptions/accounts/$accountId").map(_.subscriptions)
+
   def subscribe(subscribeRequest: SubscribeRequest): Future[List[SubscribeResponseAccount]] =
     post[List[SubscribeResponseAccount]](s"action/subscribe", subscribeRequest.asJson)
```

Comment:
Not part of this PR, but there's no need for the string interpolation here (`s""`)